website/src/app/kb/use-cases/readme.mdx

```diff
@@ -21,11 +21,11 @@ a Site.
 
 - [Block malicious DNS](/kb/use-cases/secure-dns): Use Firezone to improve your
   team's Internet security by blocking DNS queries to known malicious domains.
-- [Scale access to a VPC](/kb/use-cases/scale-vpc-access): Scale access to a
-  Google Cloud VPC using Terraform to deploy multiple Gateways.
-- [Route through a public IP](/kb/use-cases/nat-gateway): Route some of your team's
-  traffic through a single, static IP address to use services that require
-  source traffic to come from specified IP allowlists.
+- [Scale access to a VPC](/kb/use-cases/scale-vpc-access): Scale access into
+  your VPC using multiple deploys with optional Terraform example.
```

Comment:
Was the word `deploys` meant to be `gateways`?

Reply:
👍